src/schema/v2/filterArtworksConnection.ts

```diff
@@ -382,82 +394,95 @@ const filterArtworksConnectionType = connectionDefinitions({
 }).connectionType
 
+const convertFilterArgs = ({
+  additionalGeneIDs,
+  aggregationPartnerCities,
+  artistID,
+  artistIDs,
+  artistNationalities,
+  artistSeriesID,
+  atAuction,
+  attributionClass,
+  sizes,
+  dimensionRange,
+  excludeArtworkIDs,
+  extraAggregationGeneIDs,
+  includeArtworksByFollowedArtists,
+  includeMediumFilterInAggregation,
+  inquireableOnly,
+  forSale,
+  geneID,
+  geneIDs,
+  majorPeriods,
+  materialsTerms,
+  partnerID,
+  partnerIDs,
+  partnerCities,
+  priceRange,
+  saleID,
+  tagID,
+  keywordMatchExact,
+  locationCities,
+  ..._options
+}) => {
+  return {
+    additional_gene_ids: additionalGeneIDs,
+    aggregation_partner_cities: aggregationPartnerCities,
+    artist_id: artistID,
+    artist_ids: artistIDs,
+    artist_nationalities: artistNationalities,
+    artist_series_id: artistSeriesID,
+    at_auction: atAuction,
+    attribution_class: attributionClass,
+    sizes: sizes,
+    dimension_range: dimensionRange,
+    exclude_artwork_ids: excludeArtworkIDs,
+    extra_aggregation_gene_ids: extraAggregationGeneIDs,
+    include_artworks_by_followed_artists: includeArtworksByFollowedArtists,
+    include_medium_filter_in_aggregation: includeMediumFilterInAggregation,
+    inquireable_only: inquireableOnly,
+    for_sale: forSale,
+    gene_id: geneID,
+    gene_ids: geneIDs,
+    major_periods: majorPeriods,
+    materials_terms: materialsTerms,
+    partner_id: partnerID,
+    partner_ids: partnerIDs,
+    partner_cities: partnerCities,
+    price_range: priceRange,
+    sale_id: saleID,
+    tag_id: tagID,
+    keyword_match_exact: keywordMatchExact,
+    location_cities: locationCities,
+    ..._options,
+  }
+}
+
 const filterArtworksConnectionTypeFactory = (
   mapRootToFilterParams
 ): GraphQLFieldConfig<any, ResolverContext> => ({
   type: filterArtworksConnectionType,
   description: "Artworks Elastic Search results",
-  args: pageable(filterArtworksArgs),
+  args: pageableFilterArtworksArgsWithInput,
   resolve: (
     root,
-    {
-      additionalGeneIDs,
-      aggregationPartnerCities,
-      artistID,
-      artistIDs,
-      artistNationalities,
-      artistSeriesID,
-      atAuction,
-      attributionClass,
-      sizes,
-      dimensionRange,
-      excludeArtworkIDs,
-      extraAggregationGeneIDs,
-      includeArtworksByFollowedArtists,
-      includeMediumFilterInAggregation,
-      inquireableOnly,
-      forSale,
-      geneID,
-      geneIDs,
-      majorPeriods,
-      materialsTerms,
-      partnerID,
-      partnerIDs,
-      partnerCities,
-      priceRange,
-      saleID,
-      tagID,
-      keywordMatchExact,
-      locationCities,
-      ..._options
-    },
+    { input, ..._args },
     {
       unauthenticatedLoaders: { filterArtworksLoader: loaderWithCache },
       authenticatedLoaders: { filterArtworksLoader: loaderWithoutCache },
     },
     info
   ) => {
+    const argsProvidedAtRoot = convertFilterArgs(_args as any)
```

Comment:
This is the 'meat' of the change. We allow filter arguments to still come in the root, but also allow them to come in captured in `input`. Additionally, we need to convert them to the proper case, and then merge them together. When merging, we prefer `input`, and clean the data of empty values to avoid incorrect merging.